aidies-project/client/src/components/CampaignBrowser.js

```diff
@@ -0,0 +1,48 @@
+import React, { useState, useEffect } from 'react';
+import { Redirect, Link } from 'react-router-dom';
+
+import {getCampaigns} from '../store/campaign';
+import { useSelector, useDispatch } from 'react-redux';
+
+const CampaignBrowser = () => {
+    const campaigns = useSelector(state => state.campaign.list );
+    const dispatch = useDispatch();
+    useEffect(() => {
+        dispatch(getCampaigns())
+    }, [dispatch]);
```

Comment:
You actually don't need to have the `dispatch` in the dependency array. Check out this [Stack Overflow](https://stackoverflow.com/questions/54930197/react-hooks-dispatch-action-from-useeffect) for more!

Reply:
Yeah, I saw that article. left-over dependency XD